NuRadioMC/utilities/earth_attenuation.py

```diff
@@ -53,10 +70,11 @@ def get_simple_weight(theta_nu, pnu, cross_section_type = 'ghandi'):
     if(theta_nu <= 0.5 * np.pi):  # coming from above
         return np.ones_like(theta_nu)
     else:  # coming from below
-        sigma = cross_sections.get_nu_cross_section(pnu, flavors = 0, cross_section_type = cross_section_type)
-        d = - 2 * R_earth * np.cos(theta_nu)
+        sigma = cross_sections.get_nu_cross_section(pnu, flavors=0, cross_section_type=cross_section_type)
+        d = -2 * R_earth * np.cos(theta_nu)
         return np.exp(-d * sigma * DensityCRUST / AMU)
 
+
 def get_core_mantle_crust_weight(theta_nu, pnu, flavors, cross_section_type='ctw'):
```

Comment:
If I believe our own documentation core_mantle crust 1 and 2 should be identical since they are both supposed to copy ARAsim. This will just cause confusion futher down the road. What are the differences apart from Ben's code structure?